### ISSUE TITLE
Replace .reshape(-1,) with .ravel() for efficient flattening in PyTorch

### ISSUE THREAD

**Issue by SaFE-APIOpt** [JacobYuan7/DIN_GAR#26]
https://github.com/JacobYuan7/DIN-Group-Activity-Recognition-Benchmark/blob/4eceee083128ed30caf1565cfe1a6b01428225a6/train_net.py#L373
Current code:
`actions_in = torch.cat(actions_in_nopad, dim=0).reshape(-1,)  # ALL_N`
Proposed replacement:
`actions_in = torch.cat(actions_in_nopad, dim=0).ravel()  # ALL_N`
In PyTorch, both .reshape(-1,) and .ravel() flatten tensors into 1-D.

Performance: .ravel() is generally faster because it avoids additional shape checks and directly returns a contiguous view whenever possible.

Robustness: .ravel() guarantees a view if the memory is contiguous, falling back to a copy only when needed, whereas .reshape(-1,) may involve extra logic and possible reallocation.
